tests/run-make/reset-codegen-1/rmake.rs

```diff
@@ -28,11 +28,17 @@ fn main() {
         ("link-output", Some("link")),
         ("obj-output", Some("obj")),
         ("dep-output", Some("dep-info")),
-        ("multi-output", Some("asm,obj")),
     ];
     for (output_file, emit) in flags {
-        fs::remove_file(output_file).unwrap_or_default();
+        std::fs::remove_file(rmake_out_path(output_file)).unwrap_or_default();
         compile(output_file, emit);
-        fs::remove_file(output_file);
+        fs_wrapper::remove_file(rmake_out_path(output_file));
```

Comment:
Problem: unfortunate this is wrong for `("foo-output", None)` on Windows platforms because when you don't pass an `--emit=...` flag you get the binary file, which is... `bin_name(output_file)` on Windows.